### PR TITLE
Documentation

### LINKED ISSUE #7: Document progress
Ensure the GH pages has been updated with our progress.

Check documentation has been made

### PR DESCRIPTION
**Pull request by Andrew-Ritchie**
# Details

- Added Welcome page to GitHub pages.
- Added About Us page to GitHub pages.
- Added Wireframe screenshots to GitHub pages.
- Added information to the README.

# Screenshots

## Welcome Page

ScreenShot
Screenshot
Screenshot

## About Us page

Screenshot

# Checklist

## Code
- [ ] I have added some tests
- [ ] I have run the tests locally to ensure they all pass

## Documentation
- [x] I have updated documentation appropriately (GitHub Pages)
- [x] I have tested the updated GitHub Pages locally to ensure the changes to documentation render correctly (see this [Guidance](https://ese-peasy.github.io/PosturePerfection/guidance.html) page)

## Accessibility
- [x] Tested Accessibility on **Android**
  - [ ] Any added fields can be read aloud by the screen reader
- [x] Tested Accessibility on **iOS**
  - [ ] Any added fields can be read aloud by the screen reader

Closes #7

### CODE REVIEW

**Review by C-Begley — DISMISSED** [review]
I am happy to merge. Looks good

File: docs/index.md [hunk 1]
```diff
@@ -7,17 +7,31 @@ title: Welcome!
...
 </div> -->
 
-[Explain who we are and why we are completing the project]
+<div align="center"> 
+  <h1 class="fw-300">NarrateMyWay</h1>
+</div>
 
-We are a group of 5th Year Electronics & Software Engineering students...
+<div align="center" class="fs-6 fw-300"> 
+  We are a group of 5th Year Electronics & Software Engineering students at the University of Glasgow, completing a project to create a navigation application for the visually impaired. Acting as audio signs giving users a better understanding of their surrounding to improve outdoor and indoor navigation.
```
Comment (miklasr) [inline docs/index.md]: > Acting as audio signs giving users a better understanding of their surrounding to improve outdoor and indoor navigation.

Change to:

Our app provides audio signage, giving users ...

File: docs/index.md [hunk 2]
```diff
@@ -7,17 +7,31 @@ title: Welcome!
...
+</div>
+
+With a large array of navigation tools currently available to the visually impaired community NarrateMyWay aims to provide the benefits of visual signs to visually impaired users. This will give users more context to their surroundings to aid navigation. This will be achieved by connecting the user's device to various Bluetooth beacons that will be placed at significant locations. Despite the technology being developed for the University of Glasgow campus, it will have the ability to be easily expanded to any location. The technologies extensibility will also be combined with the application being free and available across Andriod and IOS devices.
```
Comment (miklasr) [inline docs/index.md]: Maybe also mention that usage isn't exclusive to visually impaired users. People who are unfamiliar with a particular location may also benefit from audio narration of their surroundings.

**Review by miklasr — CHANGES_REQUESTED** [review]
Minor changes, but otherwise good

File: docs/about-us.md [hunk 3]
```diff
@@ -4,5 +4,12 @@ title: About Us
 ---
 
 # About Us
+Our development team consists of four fifth-year Electronic and Software Engineering students from the University of Glasgow. All in our final semester we are working towards our university campus and surrounding area even more accessible.
```
Comment (Ashwin-MJ) [inline docs/about-us.md]: ... we are working towards "making" our university campus ...

**Review by Ashwin-MJ — CHANGES_REQUESTED** [review]
One minor thing, otherwise looks great!

File: docs/index.md (hunk 1 above)
Comment (Andrew-Ritchie) [reply docs/index.md]: Thanks Miklas! Made the change.

File: docs/index.md (hunk 2 above)
Comment (Andrew-Ritchie) [reply docs/index.md]: Thanks, Miklas I have made this change.

File: docs/about-us.md (hunk 3 above)
Comment (Andrew-Ritchie) [reply docs/about-us.md]: good spot, thanks Ashwin!

**Review by miklasr — APPROVED** [review]
Good stuff

**Review by Ashwin-MJ — APPROVED** [review]
Nice!

**Pull request merged**